mmv1/templates/terraform/examples/privateca_certificate_authority_basic_no_org.tf.tmpl

```diff
@@ -0,0 +1,35 @@
+resource "google_privateca_certificate_authority" "{{$.PrimaryResourceId}}" {
```

Comment:
you added a new test file but didn't include it in the examples list, see [`add create test`](https://googlecloudplatform.github.io/magic-modules/test/test/#add-a-create-test) and [`add update test`](https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test)